File: src/main/java/org/evertones/persistence/modules/product/ProductRepository.java
package org.evertones.persistence.modules.product;

import org.evertones.model.modules.product.Product;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;
import org.springframework.stereotype.Repository;

@Repository
public interface ProductRepository extends JpaRepository<Product, Integer>, QueryDslPredicateExecutor<Product> {
}

File: src/main/scala/org/evertones/persistence/modules/product/ProductRepository.scala
package org.evertones.persistence.modules.product


import org.evertones.model.modules.product.Product
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QueryDslPredicateExecutor

trait ProductRepository extends JpaRepository[Product, Integer]
        with QueryDslPredicateExecutor[Product] {

}
